src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,26 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_VALID_AGE = 35;
+    private static final String VALID_NATIONALITY = "Ukrainian";
+    private static final int MIN_VALID_PERIOD = 10;
+    private static final int FROM_YEAR_INDEX = 0;
+    private static final int TO_YEAR_INDEX = 1;
```

Comment:
```suggestion
    private static final int MIN_VALID_AGE = 35;
    private static final int MIN_VALID_PERIOD = 10;
    private static final int FROM_YEAR_INDEX = 0;
    private static final int TO_YEAR_INDEX = 1;
    private static final String VALID_NATIONALITY = "Ukrainian";
```

I think you can order constants by type. I'm not sure about this exactly, but it looks more readable